2021/01/09 18:21:02 [emerg] 14124#11676: no "events" section in configuration
2021/01/09 18:21:07 [emerg] 12616#14660: no "events" section in configuration
2021/01/09 18:22:12 [emerg] 4432#9680: unknown directive "worker_process" in ./conf/nginx.conf:1
2021/01/09 19:21:39 [error] 10780#10020: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 19:21:39 [error] 10780#10020: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 19:22:39 [error] 10780#10020: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 19:22:39 [error] 10780#10020: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 19:32:27 [error] 11364#748: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 19:32:28 [error] 11364#748: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 19:33:22 [error] 11364#748: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 19:33:22 [error] 11364#748: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:22):2: module 'resty.template' not found:
	no field package.preload['resty.template']
	no file './site/lualib/resty\template.ljbc'
	no file './site/lualib/resty\template/init.ljbc'
	no file './lualib/resty\template.ljbc'
	no file './lualib/resty\template/init.ljbc'
	no file './site/lualib/resty\template.lua'
	no file './site/lualib/resty\template/init.lua'
	no file './lualib/resty\template.lua'
	no file './lualib/resty\template/init.lua'
	no file '.\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\resty\template\init.lua'
	no file './site/lualib/resty\template.so'
	no file './lualib/resty\template.so'
	no file '.\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty\template.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty\template.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
	no file './site/lualib/resty.so'
	no file './lualib/resty.so'
	no file '.\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\resty.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\resty.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:22):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:11:36 [emerg] 9160#6576: directive "set" is not terminated by ";" in ./conf/nginx.conf:12
2021/01/09 20:13:02 [error] 7560#9552: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:02 [error] 7560#9552: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:02 [error] 7560#9552: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:02 [error] 7560#9552: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:11 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:13:12 [error] 2828#4036: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:56 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:56 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:56 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:56 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:57 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:16:58 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:03 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:03 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:03 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:03 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:04 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:05 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:17:06 [error] 2828#4036: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:29):7: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:07 [error] 1152#12420: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:07 [error] 1152#12420: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:07 [error] 1152#12420: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:07 [error] 1152#12420: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:08 [error] 1152#12420: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:08 [error] 1152#12420: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:08 [error] 1152#12420: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:08 [error] 1152#12420: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:20 [error] 12700#10532: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:20 [error] 12700#10532: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:20 [error] 12700#10532: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:20 [error] 12700#10532: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:19:21 [error] 12700#10532: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):9: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:21:21 [error] 14292#10916: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:32):2: module 'test' not found:
	no field package.preload['test']
	no file '%prefix/lua/test.lua'
	no file './site/lualib/test.ljbc'
	no file './site/lualib/test/init.ljbc'
	no file './lualib/test.ljbc'
	no file './lualib/test/init.ljbc'
	no file './site/lualib/test.lua'
	no file './site/lualib/test/init.lua'
	no file './lualib/test.lua'
	no file './lualib/test/init.lua'
	no file '.\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test\init.lua'
	no file './site/lualib/test.so'
	no file './lualib/test.so'
	no file '.\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:32):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:21:22 [error] 14292#10916: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:32):2: module 'test' not found:
	no field package.preload['test']
	no file '%prefix/lua/test.lua'
	no file './site/lualib/test.ljbc'
	no file './site/lualib/test/init.ljbc'
	no file './lualib/test.ljbc'
	no file './lualib/test/init.ljbc'
	no file './site/lualib/test.lua'
	no file './site/lualib/test/init.lua'
	no file './lualib/test.lua'
	no file './lualib/test/init.lua'
	no file '.\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test\init.lua'
	no file './site/lualib/test.so'
	no file './lualib/test.so'
	no file '.\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:32):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:21:33 [error] 14292#10916: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:32):2: module 'test' not found:
	no field package.preload['test']
	no file '%prefix/lua/test.lua'
	no file './site/lualib/test.ljbc'
	no file './site/lualib/test/init.ljbc'
	no file './lualib/test.ljbc'
	no file './lualib/test/init.ljbc'
	no file './site/lualib/test.lua'
	no file './site/lualib/test/init.lua'
	no file './lualib/test.lua'
	no file './lualib/test/init.lua'
	no file '.\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test\init.lua'
	no file './site/lualib/test.so'
	no file './lualib/test.so'
	no file '.\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:32):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:21:33 [error] 14292#10916: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:32):2: module 'test' not found:
	no field package.preload['test']
	no file '%prefix/lua/test.lua'
	no file './site/lualib/test.ljbc'
	no file './site/lualib/test/init.ljbc'
	no file './lualib/test.ljbc'
	no file './lualib/test/init.ljbc'
	no file './site/lualib/test.lua'
	no file './site/lualib/test/init.lua'
	no file './lualib/test.lua'
	no file './lualib/test/init.lua'
	no file '.\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test\init.lua'
	no file './site/lualib/test.so'
	no file './lualib/test.so'
	no file '.\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:32):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:23:28 [error] 13792#16448: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:32):2: module 'test' not found:
	no field package.preload['test']
	no file './lua/test.lua'
	no file './site/lualib/test.ljbc'
	no file './site/lualib/test/init.ljbc'
	no file './lualib/test.ljbc'
	no file './lualib/test/init.ljbc'
	no file './site/lualib/test.lua'
	no file './site/lualib/test/init.lua'
	no file './lualib/test.lua'
	no file './lualib/test/init.lua'
	no file '.\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test\init.lua'
	no file './site/lualib/test.so'
	no file './lualib/test.so'
	no file '.\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:32):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:23:28 [error] 13792#16448: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:32):2: module 'test' not found:
	no field package.preload['test']
	no file './lua/test.lua'
	no file './site/lualib/test.ljbc'
	no file './site/lualib/test/init.ljbc'
	no file './lualib/test.ljbc'
	no file './lualib/test/init.ljbc'
	no file './site/lualib/test.lua'
	no file './site/lualib/test/init.lua'
	no file './lualib/test.lua'
	no file './lualib/test/init.lua'
	no file '.\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test.lua'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lua\test\init.lua'
	no file './site/lualib/test.so'
	no file './lualib/test.so'
	no file '.\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\lualib\test.so'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\test.dll'
	no file 'D:\Users\Sledmine\Documents\Developing\openresty-1.19.3.1-win64\loadall.dll'
stack traceback:
coroutine 0:
	[C]: in function 'require'
	content_by_lua(nginx.conf:32):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:23:40 [error] 10672#16556: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:23:40 [error] 10672#16556: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:31):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:24:13 [error] 8120#7976: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:13 [error] 8120#7976: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:24:13 [error] 8120#7976: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:14 [error] 8120#7976: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:24:14 [error] 8120#7976: *5 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:14 [error] 8120#7976: *6 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:14 [error] 8120#7976: *7 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:14 [error] 8120#7976: *8 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:24:49 [error] 8696#15608: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:50 [error] 8696#15608: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:24:50 [error] 8696#15608: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:51 [error] 8696#15608: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:24:56 [error] 8696#15608: *5 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:24:56 [error] 8696#15608: *6 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:25:00 [error] 8696#15608: *7 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:25:00 [error] 8696#15608: *8 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:25:00 [error] 8696#15608: *9 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:25:00 [error] 8696#15608: *10 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:25:24 [error] 6292#12604: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:25:25 [error] 6292#12604: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:25:29 [error] 6292#12604: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:25:29 [error] 6292#12604: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:25:29 [error] 6292#12604: *5 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:25:29 [error] 6292#12604: *6 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:25:30 [error] 6292#12604: *7 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:31):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:31):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:26:28 [error] 1652#11832: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:29):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:29):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:26:28 [error] 1652#11832: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:29):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:29):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:27:22 [error] 3440#9256: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:29):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:29):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:27:22 [error] 3440#9256: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:29):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:29):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:27:23 [error] 3440#9256: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:29):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:29):4: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:27:23 [error] 3440#9256: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:29):4: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:29):4: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:28:06 [error] 10000#16320: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:28:06 [error] 10000#16320: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:27):3: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:30:52 [error] 6596#14948: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:30:52 [error] 6596#14948: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:25):3: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:31:34 [error] 11616#15804: *1 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:26):2: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:26):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:31:34 [error] 11616#15804: *2 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:26):2: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:26):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:31:35 [error] 11616#15804: *3 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:26):2: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:26):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:31:35 [error] 11616#15804: *4 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:26):2: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:26):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:31:41 [error] 11616#15804: *5 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:26):2: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:26):2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/09 20:31:42 [error] 11616#15804: *6 lua entry thread aborted: runtime error: content_by_lua(nginx.conf:26):2: bad argument #1 to 'say' (string, number, boolean, nil, ngx.null, or array table expected, but got function)
stack traceback:
coroutine 0:
	[C]: in function 'say'
	content_by_lua(nginx.conf:26):2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:33:35 [error] 14792#12896: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:33:35 [error] 14792#12896: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:26):3: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:44 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:45 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/09 20:39:46 [error] 10088#15392: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:29:27 [error] 10088#15392: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:29:27 [error] 10088#15392: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:29:27 [error] 10088#15392: *3 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:29:27 [error] 10088#15392: *3 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	content_by_lua(nginx.conf:30):6: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:29:58 [emerg] 17312#6768: unexpected "}" in ./conf/nginx.conf:24
2021/01/10 01:30:13 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:13 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:13 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:13 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:20 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:20 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:20 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:20 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:21 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:46 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:46 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:46 [error] 13772#12868: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:46 [error] 13772#12868: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:52 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:52 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:53 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:53 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:30:53 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:30:53 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:14 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:14 [error] 8304#12568: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:23 [error] 9972#5576: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:23 [error] 9972#5576: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:24 [error] 9972#5576: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:24 [error] 9972#5576: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:51 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:51 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:31:53 [error] 16724#3940: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:26 [error] 5640#14808: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:26 [error] 5640#14808: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:27 [error] 5640#14808: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:27 [error] 5640#14808: *1 CreateFile() "./.templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:41 [error] 10144#14892: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:41 [error] 10144#14892: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:57 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:57 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:58 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:58 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:58 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:58 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:58 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:58 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:33:59 [error] 11940#5244: *1 lua entry thread aborted: runtime error: ./lua/main.lua:2: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:36:52 [error] 4060#9616: *1 CreateFile() "./../templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "./templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:36:52 [error] 4060#9616: *1 CreateFile() "./../templates/packagesIndex.html" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", subrequest: "./templates/packagesIndex.html", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:38:20 [error] 9492#16340: *1 lua subrequests cycle while processing "/templates/packagesIndex.html", client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:38:20 [error] 9492#16340: *1 lua entry thread aborted: runtime error: ...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: failed to issue subrequest: -1
stack traceback:
coroutine 0:
	[C]: in function 'capture'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:162: in function 'load'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:405: in function 'parse'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:389: in function 'compile'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:637: in function 'process'
	...veloping\openresty-1.19.3.1-win64\lua\resty\template.lua:652: in function 'render'
	./lua/main.lua:2: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", subrequest: "/templates/packagesIndex.html", host: "localhost:8080"
2021/01/10 01:49:36 [error] 13672#7428: *1 CreateFile() "./html/favicon.ico" failed (2: El sistema no puede encontrar el archivo especificado), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 01:49:39 [error] 13672#7428: *1 CreateFile() "./html/templates/templates/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /templates/templates/css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:49:39 [error] 13672#7428: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:49:39 [error] 13672#7428: *1 CreateFile() "./html/assets/images/favicon.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/favicon.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:49:51 [error] 13672#7428: *1 CreateFile() "./html/templates/templates/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /templates/templates/css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:49:51 [error] 13672#7428: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:51:35 [error] 11508#3924: *1 CreateFile() "./html/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:51:35 [error] 11508#3924: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:51:53 [error] 11508#3924: *1 CreateFile() "./html/templates/templates/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /templates/templates/css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:51:53 [error] 11508#3924: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:51:55 [error] 11508#3924: *1 CreateFile() "./html/templates/templates/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /templates/templates/css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:51:56 [error] 11508#3924: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/templates/"
2021/01/10 01:53:29 [error] 11508#3924: *3 CreateFile() "./html/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:53:29 [error] 11508#3924: *3 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:54:11 [error] 7404#4800: *1 CreateFile() "./html/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:54:11 [error] 7404#4800: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:54:25 [error] 7404#4800: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:54:26 [error] 7404#4800: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:54:37 [error] 7404#4800: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:54:46 [error] 7404#4800: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:56:49 [error] 14308#14916: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:56:58 [error] 14308#14916: *1 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 01:58:53 [error] 14308#14916: *3 CreateFile() "./html/assets/images/mercury.png" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /assets/images/mercury.png HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/mercury"
2021/01/10 02:06:56 [error] 1152#13724: *1 directory index of "./html/" is forbidden, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/10 02:07:00 [error] 1152#13724: *1 directory index of "./html/css/" is forbidden, client: 127.0.0.1, server: , request: "GET /css/ HTTP/1.1", host: "localhost:8080"
2021/01/10 02:07:44 [emerg] 10036#2776: CreateFile() "./conf/.conf/mime.types" failed (3: El sistema no puede encontrar la ruta especificada) in ./conf/nginx.conf:8
2021/01/10 02:07:56 [emerg] 5164#1148: CreateFile() "./conf/conf/mime.types" failed (3: El sistema no puede encontrar la ruta especificada) in ./conf/nginx.conf:8
2021/01/10 02:08:13 [emerg] 9720#9088: CreateFile() "./conf/conf/mime.types" failed (3: El sistema no puede encontrar la ruta especificada) in ./conf/nginx.conf:8
2021/01/10 15:22:01 [error] 16404#488: *1 lua entry thread aborted: runtime error: ./lua/main.lua:6: attempt to index local 'f' (a nil value)
stack traceback:
coroutine 0:
	./lua/main.lua: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/10 15:22:01 [error] 16404#488: *2 lua entry thread aborted: runtime error: ./lua/main.lua:6: attempt to index local 'f' (a nil value)
stack traceback:
coroutine 0:
	./lua/main.lua: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/10 15:22:09 [error] 16404#488: *3 lua entry thread aborted: runtime error: ./lua/main.lua:6: attempt to index local 'f' (a nil value)
stack traceback:
coroutine 0:
	./lua/main.lua: in main chunk, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/10 15:22:10 [error] 16404#488: *4 lua entry thread aborted: runtime error: ./lua/main.lua:6: attempt to index local 'f' (a nil value)
stack traceback:
coroutine 0:
	./lua/main.lua: in main chunk, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/16 20:35:17 [error] 7756#4208: *1 CreateFile() "./html/css/skeleton.css" failed (3: El sistema no puede encontrar la ruta especificada), client: 127.0.0.1, server: , request: "GET /css/skeleton.css HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/16 20:40:07 [error] 11192#10132: *1 directory index of "./html/" is forbidden, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/16 20:40:08 [error] 11192#10132: *1 CreateFile() "./html/favicon.ico" failed (2: El sistema no puede encontrar el archivo especificado), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/16 20:40:20 [error] 11192#10132: *1 directory index of "./html/" is forbidden, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/18 02:46:58 [error] 6952#6544: *1 directory index of "./html/" is forbidden, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/18 02:46:58 [error] 6952#6544: *1 CreateFile() "./html/favicon.ico" failed (2: El sistema no puede encontrar el archivo especificado), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
2021/01/18 02:47:08 [error] 6952#6544: *1 directory index of "./html/" is forbidden, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/18 02:47:08 [error] 6952#6544: *1 directory index of "./html/" is forbidden, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", host: "localhost:8080"
2021/01/18 02:50:30 [error] 2736#11548: *1 directory index of "./html/img/" is forbidden, client: 127.0.0.1, server: , request: "GET /img/ HTTP/1.1", host: "localhost:8080"
2021/01/18 02:50:30 [error] 2736#11548: *1 CreateFile() "./html/favicon.ico" failed (2: El sistema no puede encontrar el archivo especificado), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/img/"
2021/01/18 02:50:46 [error] 14256#15652: *1 directory index of "./html/img/" is forbidden, client: 127.0.0.1, server: , request: "GET /img/ HTTP/1.1", host: "localhost:8080"
2021/01/18 02:50:47 [error] 14256#15652: *1 directory index of "./html/img/" is forbidden, client: 127.0.0.1, server: , request: "GET /img/ HTTP/1.1", host: "localhost:8080"
2021/01/18 02:50:58 [error] 14256#15652: *6 directory index of "./html/img/" is forbidden, client: 127.0.0.1, server: , request: "GET /img/ HTTP/1.1", host: "localhost:8080"
2021/01/18 02:50:59 [error] 14256#15652: *6 CreateFile() "./html/favicon.ico" failed (2: El sistema no puede encontrar el archivo especificado), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/img/"
2021/01/18 02:51:11 [error] 568#2352: *1 directory index of "./html/img/" is forbidden, client: 127.0.0.1, server: , request: "GET /img/ HTTP/1.1", host: "localhost:8080"
2021/01/18 02:51:31 [error] 7644#3548: *2 CreateFile() "./html/favicon.ico" failed (2: El sistema no puede encontrar el archivo especificado), client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/img/"
